src/components/webview/sourceControl.ts

```diff
@@ -48,7 +48,7 @@ export class SourceControlService {
 		this.__messageBus.publish({ kind: MessageKind.beforePRCreated });
 
 		const result = await axios.post<CreatePRResponse>(
-			'https://telemetry.intuita.io/sourceControl/github/pulls',
+			'https://codemod-studio.vercel.app/sourceControl/github/pulls',
```

Comment:
I think this was correct, am I right?

Reply:
i've already reverted that change. At first i thought that we moved our backend to the new host